src/write.c

```diff
@@ -2185,11 +2185,24 @@ avifResult avifEncoderFinish(avifEncoder * encoder, avifRWData * output)
     }
 #endif
 
+    // According to section 5.2 of AV1 Image File Format specification v1.1.0:
+    //   If the the primary item or all the items referenced by the primary item are AV1 image items made only
+    //   of Intra Frames, the brand "avio" should be used in the compatible_brands field of the FileTypeBox.
+    // See https://aomediacodec.github.io/av1-avif/v1.1.0.html#image-and-image-collection-brand.
+    // Use the "avio" brand in all cases except for layered images, because:
+    //  - Non-layered still images are always Intra Frames, even with grids;
+    //  - Sequences cannot be combined with layers or grids, and the first frame of the sequence
+    //    (referred to by the primary image item) is always an Intra Frame.
+    const avifBool useAvioBrand = encoder->extraLayerCount == 0;
```

Comment:
IMPORTANT: Please see the additional requirement for the avio brand for image sequences:

https://aomediacodec.github.io/av1-avif/v1.1.0.html#image-sequence-brand

> ... at least one of the [AV1 image sequences](https://aomediacodec.github.io/av1-avif/v1.1.0.html#av1-image-sequence) shall be made only of [AV1 Samples](https://aomediacodec.github.io/av1-avif/v1.1.0.html#biblio-av1-isobmff) marked as [sync](https://aomediacodec.github.io/av1-avif/v1.1.0.html#biblio-isobmff).

Reply:
Done.